src/components/parcourslist/ParcoursList.jsx

```diff
@@ -1,40 +1,49 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
 import * as _ from 'lodash';
 import Parcours from '../parcours/Parcours';
-import Leaflet from '../map/Map';
 import { ApiContext } from '../../contexts/ApiContext';
 import '../../App.scss';
 import './ParcoursList.scss';
+import MapParcours from '../map/MapParcours';
+import CurrentCourseContext from '../../contexts/CurrentCourseContext';
 
 const ParcoursList = () => {
   const { course } = useContext(ApiContext);
   const parcours = [...course];
+  const [currentCourse, setCurrentCourse] = useState(1);
 
   const uniqueCourseByID = _.uniqBy(parcours, 'course_id');
 
```

Comment:
Initializing currentCourse with "useState(1)" ? Why not 0 ? Isn't it safer ?

Reply:
Because the state is compared to id courses. So i put an initial state to 1 to display the first course on the map by default.

Reply:
Yeah, ok, It's based on SQL id, got it.